packages/app/obojobo-repository/shared/components/repository-nav.jsx

```diff
@@ -49,7 +86,16 @@ const RepositoryNav = props => {
 				{props.userId !== 0 ? (
 					<div className="repository--nav--current-user">
 						<button onClick={onToggleMenu}>
-							<div className="repository--nav--current-user--name">{props.displayName}</div>
+							<div className="repository--nav--current-user--name">
```

Comment:
This doesn't appear to be reachable in the tab order. I'm not sure how the rest of the page is handling tab order, but the notifications indicator doesn't appear to be part of it.